store/sqlstore/upgrade.go

```diff
@@ -341,7 +341,7 @@ func upgradeDatabaseToVersion35(sqlSupplier *SqlSupplier) {
 		sqlSupplier.GetMaster().Exec("UPDATE ChannelMembers SET Roles = 'channel_user channel_admin' WHERE Roles = 'admin'")
 
 		// The rest of the migration from Filenames -> FileIds is done lazily in api.GetFileInfosForPost
-		sqlSupplier.CreateColumnIfNotExists("Posts", "FileIds", "varchar(150)", "varchar(150)", "[]")
+		sqlSupplier.CreateColumnIfNotExists("Posts", "FileIds", "varchar(300)", "varchar(300)", "[]")
```

Comment:
This line should be kept as is since it's about a previous version. Our DB schema migration system works by defining a function for each new version. 

What we want to do is add a line performing the type change in the function for the next version, currently `upgradeDatabaseToVersion530`:

```go
sqlSupplier.AlterColumnTypeIfExists("Posts", "FileIds", "text", "varchar(300)")
```

There's a small caveat here which is we need to define a `text` type for MySQL instead of `varchar(300)`. This is because our underlying ORM `gorp` implicitly converts any size above 255 as a `text` on MySQL.

Reply:
My bad :sweat_smile: should have scrolled a bit longer. 
Thanks for the introduction to Mattermost DB schema migration :) should be good now.